lms/static/js/discovery/views/refine_sidebar.js

```diff
@@ -44,11 +44,13 @@
             },
 
             renderFacet: function(facetKey, options) {
+                var optionsSelected = _.filter(options, function(o) { return o.get('selected'); });
+                var listIsHuge = optionsSelected.length ? false : (options.length > 9);
```

Comment:
Shouldn't it be the other way around? I thought optionsSelected.length evaluates to true if it is zero or a hundred. And if it is true, then we think the list is not huge...